### ISSUE TITLE
Unable to access 'Dashboard' - TypeError: this.state.dashboards.map is not a function

### ISSUE THREAD

**Issue by QuentinMoss** [influxdata/chronograf#743]
OS: CentOS 7.2
RPM Download Link: https://dl.influxdata.com/chronograf/releases/chronograf-1.1.0~beta5.x86_64.rpm
Docker Container: quay.io/influxdb/chronograf:latest

I have downloaded Chronograf Beta v4, Beta v5, and the latest Docker container and each one has given me the following error when loading the "Dashboards" page.

```
http://localhost:8081/chronograf/v1/me Failed to load resource: the server responded with a status of 418 (I'm a teapot)

app.b4e1c48….js:4 Uncaught (in promise) TypeError: this.state.dashboards.map is not a function
    at Object.render (app.b4e1c48….js:4)
    at d._renderValidatedComponentWithoutOwnerOrContext (vendor.6a8eed0….js:191)
    at d._renderValidatedComponent (vendor.6a8eed0….js:191)
    at d._updateRenderedComponent (vendor.6a8eed0….js:191)
    at d._performComponentUpdate (vendor.6a8eed0….js:191)
    at d.updateComponent (vendor.6a8eed0….js:191)
    at d.performUpdateIfNecessary (vendor.6a8eed0….js:191)
    at Object.performUpdateIfNecessary (vendor.6a8eed0….js:38)
    at s (vendor.6a8eed0….js:38)
    at r.perform (vendor.6a8eed0….js:40)
    at i.perform (vendor.6a8eed0….js:40)
    at i.perform (vendor.6a8eed0….js:38)
    at Object.T [as flushBatchedUpdates] (vendor.6a8eed0….js:38)
    at r.closeAll (vendor.6a8eed0….js:40)
    at r.perform (vendor.6a8eed0….js:40)
    at Object.batchedUpdates (vendor.6a8eed0….js:192)
    at Object.u [as enqueueUpdate] (vendor.6a8eed0….js:38)
    at r (vendor.6a8eed0….js:41)
    at Object.enqueueSetState (vendor.6a8eed0….js:41)
    at Object.r.setState (vendor.6a8eed0….js:55)
    at app.b4e1c48….js:4
```

**Comment by goller** [comment]
@QuentinMoss Yikes, looking at it now.

**Comment by nhaugo** [comment]
@QuentinMoss the dashboards section doesn't actually do anything at the moment. So you aren't missing anything there. We will be rolling more out around that in the next on an upcoming release.

**Comment by nhaugo** [comment]
Added this to the beta5 release notes

**Comment by QuentinMoss** [comment]
@nhaugo Thanks.

**Comment by jademcgough** [comment]
This can be closed when the work for #614 is merged.

**Comment by nhaugo** [comment]
Not quite a dupe but as @jademcgough will be fixed by #614
